<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Обучение</string>
   </property>
   <widget class="QPushButton" name="bOpenFileTrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Обучить из файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="lStatusTrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Повторять</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Тестирование 1</string>
   </property>
   <widget class="QLabel" name="lResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bGetChar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Получть ответ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bClear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRes">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ожидаемый результат</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Нейросеть 1</string>
   </property>
   <widget class="QLabel" name="lName1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название: </string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Тестирование 2</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Обучать если не верно</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bOpenFileTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл тестирования</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Нейросеть 1</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lTrue1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Верно - </string>
     </property>
    </widget>
    <widget class="QLabel" name="lFalse1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Неверно - </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Нейросеть 2</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lTrue2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Верно - </string>
     </property>
    </widget>
    <widget class="QLabel" name="lFalse2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Неверно - </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bTest1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Тестировать сеть 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCountExamples">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество примеров - 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bTest2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Тестировать сеть 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lStatusTest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Нейросеть 2</string>
   </property>
   <widget class="QLabel" name="lName2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="bOpen2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
